src/fidesops/service/connectors/saas_connector.py

```diff
@@ -60,6 +61,9 @@ def add_authentication(
         elif strategy == "bearer_authentication":
             token_key = pydash.get(configuration, "token.connector_param")
             req.headers["Authorization"] = "Bearer " + self.secrets[token_key]
+        elif strategy == "query_param":
+            token_key = pydash.get(configuration, "token.connector_param")
+            req.url = set_query_parameter(req.url, token_key, self.secrets[token_key])
```

Comment:
Interesting auth method that we've got to support -

Reply:
This is a specific instance of a more general auth strategy we've scoped out for later
[[SaaS Connector] API Key authentication strategy #334](https://github.com/ethyca/fidesops/issues/334)